### PR TITLE
feat(repo): shared ui lib

### PR DESCRIPTION
**Pull request by bearni95**
- [x] Svelte library for `ui-lib` package
- [x] Storybook for showcasing components
- [x] Tailwind styling and core classes
- [x] Implement Taiko's core colors

### Components
- [x] `ResponsiveController`
- [x] Footer
- [X] Icons

### CODE REVIEW

File: .github/workflows/repo--validate-pr-title.yml [hunk 1]
```diff
@@ -41,6 +41,7 @@ jobs:
             protocol
             relayer
             taikoon
+            ui-lib
```
Comment (KorbinianK) [inline .github/workflows/repo--validate-pr-title.yml:44]: Let's move the ui lib setup to a separate PR that gets merged before the actual ui-lib package. Otherwise it will show up in the taikoon changelogs.

File: packages/taikoon-ui/src/components/sections/Footer.section.svelte [hunk 2]
```diff
@@ -1,261 +1,17 @@
 <script lang="ts">
-  import { json, t } from 'svelte-i18n';
+  import { Footer } from 'ui-lib';
```
Comment (KorbinianK) [inline packages/taikoon-ui/src/components/sections/Footer.section.svelte:2]: Let's not change multiple packages in a single PR, even though they are related. 

Split this PR into 3 to keep everything organized.

1. ui-lib repo setup (workflow, pr title)
2. ui-lib package add
3. taikoon package changes

**Review by KorbinianK — CHANGES_REQUESTED** [review]

File: .github/workflows/repo--validate-pr-title.yml (hunk 1 above)
Comment (bearni95) [reply .github/workflows/repo--validate-pr-title.yml:44]: https://github.com/taikoxyz/taiko-mono/pull/17219

**Review by KorbinianK — APPROVED** [review]

**Review by bennettyong — APPROVED** [review]

**Review by bennettyong — APPROVED** [review]

**Review by bennettyong — APPROVED** [review]
